src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,127 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table = new Node[INITIAL_CAPACITY];
+    private int size;
+
+    private static class Node<K,V> {
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
```

Comment:
The checklist points out that you should not use getters and setters in the Node class. As we can see, you're not using them, which is good. However, you still have a public constructor in the Node class. It's not necessary, as the Node class is private and won't be used outside of the MyHashMap class. You can make the constructor package-private (by removing the 'public' keyword) or even private, because you only instantiate Node objects within the MyHashMap class.